.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,45 +1,29 @@
-<!--- Provide a general summary of your changes in the Title above -->
+<!--- Title: Provide a general summary of your changes in the Title above -->
+<!--- Reviewers: Include @karstens, @GarthDB, @mrcjhicks, @lynnhao -->
 
 ## Description
 
-<!--- Describe your changes in detail -->
+<!--- Describe your changes in detail, including a list of changes -->
 
 ## Related Issue
 
 <!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If suggesting a new feature or change, please discuss it in #spectrum_tokens_talk or design workshop, first -->
 <!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
-
-## Motivation and Context
```

Comment:
Suggesting to remove this section since I feel the "Description" would be sufficient for now.

Reply:
@lynnhao Would the description also include the design motivation for the change? I ask because @GarthDB could use that information to in his tokens release notes. That could be helpful.

Reply:
I like having the design motivation for changes. I don't mind if it's covered in the description, but I think it would be easiest to have it broken out into its own section.

Reply:
I would ask to keep it so we can describe what changes in the "Description" section and why in the "Motivation and Context" section.